[docs]: update configuration endpoints (#435)
hyperledger/iroha-2-docs: src/reference/torii-endpoints.md

sending the confirmation message, the server starts streaming all the blocks from the given block number up to the
current block and continues to stream blocks as they are added to the blockchain.

## Configuration / Retrieve

- **Protocol:** HTTP
- **Method:** `GET`
- **Endpoint:** `/configuration`

> - **Responses:** with JSON-serialized subset of configuration parameters. The subset of returned parameters is equal to

Reviewer: ```suggestion
- **Responses:** with a JSON-serialized subset of configuration parameters. The subset of returned parameters is equal to
```

>   the one accepted by [Configuration > Update endpoint](#configuration-update), i.e. it contains only

Reviewer: ```suggestion
  the one accepted by the [Configuration > Update endpoint](#configuration-update), i.e. it only contains the
```

  `logger.level` parameter as of now.

**Example response:**

```json
{
  "logger": {
    "level": "TRACE"
  }
}
```

## Configuration / Update

- **Protocol:** HTTP
- **Method:** `POST`
- **Endpoint:** `/configuration`
- **Expects:** a JSON-serialized subset of configuration parameters.
- **Response:** `202 ACCEPTED`

> This endpoint currently supports only dynamic updating of the `logger.level` parameter.

Reviewer: ```suggestion
This endpoint only supports dynamic updating of the `logger.level` parameter for now.
```

For possible values please refer to the configuration reference (TODO:
[Tracking issue for configuration reference](https://github.com/hyperledger/iroha-2-docs/issues/392)).

**Example request:**

```json
{
  "logger": {
    "level": "DEBUG"
  }
}
```

::: tip Guarantees

> It is not guaranteed that successful update of configuration means that the configuration is indeed updated. While

Reviewer: ```suggestion
A successful configuration update is not guaranteed to mean the configuration is indeed updated. While
```

Reviewer 2: ```suggestion
A successful configuration update does not guarantee that the configuration is indeed updated. While
```

      results.
    - **`sort_by_metadata_key`:** An optional parameter in queries. Use to sort results containing metadata with a given
      key.

>     - **`fetch_size`:** An optional parameter in queries. Use to specify the exact number of results returned by a

Reviewer: ```suggestion
    - **`fetch_size`:** An optional parameter in queries. Use it to specify the exact number of results returned by a
```
